contrib/pai_vscode/README.md

```diff
@@ -41,6 +41,18 @@ The extension has a useful feature called "Simulate Job Running". It enables the
 
 ![](https://raw.githubusercontent.com/Microsoft/pai/master/contrib/pai_vscode/assets/simulate-job.gif)
 
+## Sidebar
+HDFS Explorer and Job List view will be shown in the extension's view container (sidebar).
+
+### HDFS Explorer
+1. Connected to a PAI cluster's hdfs by double click the corresponding tree node.
```

Comment:
1. You are able to connect to an OpenPAI cluster's HDFS by double click its corresponding tree node.